nntrainer/src/fc_layer.cpp

```diff
@@ -168,6 +168,10 @@ Tensor FullyConnectedLayer::backwarding(Tensor derivative, int iteration) {
                   .applyIf(this->isWeightDecayL2Norm(), _LIFT(add_i), weight, weight_decay.lambda)
                   .run();
 
+  gradients.clear();
+  gradients.push_back(djdw);
+  gradients.push_back(djdb);
+
```

Comment:
how do you manage the order of this gradients if there are multiple djdw and djdb. are you going to set the order like djdw[0], djdb[0], djdw[1], djdb[1]? or djdw[0], djdw[1], djdw[2], djdb[0], djdb[1]...? the former order is more proper I think. cause apply_gradient function asks this order.

Reply:
> how do you manage the order of this gradients if there are multiple djdw and djdb. are you going to set the order like djdw[0], djdb[0], djdw[1], djdb[1]? or djdw[0], djdw[1], djdw[2], djdb[0], djdb[1]...? 

Yes this will be the order.